kubernetes/zenko/charts/backbeat/templates/api/deployment.yaml

```diff
@@ -52,6 +52,7 @@ spec:
             httpGet:
               path: /_/healthcheck
```

Comment:
Is it safer for this one to use a hard coded value vs. `.Values.health.path.liveness`?

Reply:
Oh never mind, I see they are different now.